src/components/task.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Item from './item';
+
+const Task = props => (
+  <li className="task__item">
+    {props.variety !== '' ?
```

Comment:
Yeah, I originally had the operator without any `!== ''` going on, but then I was worried about how it would handle falsy values that might end up being 'valid' from a user perspective. It tripped us up at work because stuff we were expecting to be caught by the brief and more implicit check were not caught. I can always test it out and see what happens.

With regard to default props, if you provide a PropType that is not set to `.isRequired`, you must give it a default prop value, otherwise React complains. If it's not appropes to set a default value for whatever reason, you can set it as .isRequired but then of course you have to ensure the component definitely receives a prop or it will fall over :)